python/semantic_kernel/planning/stepwise_planner/Skills/StepwiseStep/skprompt.txt

```diff
@@ -24,6 +24,18 @@ Here is an example of a valid $JSON_BLOB:
   "action": "functionName",
   "action_variables": {"parameterName": "some value", ...}
 }
+
+Here is another example of a valid $JSON_BLOB:
+{
+  "action": "_Function_.Name",
+  "action_variables": {"parameterName": "some value", ...}
+}
+
+Here is another example of a valid $JSON_BLOB:
+{
+  "action": "FunctionName2",
```

Comment:
another question is about this example.

What makes it different from the first example?

```txt
"action": "functionName"
```

Is it just the naming convention?

Reply:
Yes, capitalization was also not being respected. The LLM was "camelCasing" the function name in some instances. We want it to leave it as-is.

Reply:
That makes sense to me.